.github/workflows/pre_release.yaml

```diff
@@ -0,0 +1,90 @@
+name: Create a pre-release
+
+on:
+  # Trigger a beta version release (pre-release) on push to the master branch.
+  push:
+    branches:
+      - use-release-metadata-action # TODO
```

Comment:
how is this supposed to be working? canary builds should be done only on the default branch

Reply:
that's so that I can test the workflow against this branch, I intend to change it to `master` before merging